etl/steps/data/meadow/papers/2022-11-03/zijdeman_et_al_2015/__init__.py

```diff
@@ -0,0 +1,87 @@
+from typing import Tuple
```

Comment:
Why create an `__init__.py` file? You could just have a file `zijdeman_et_al_2015.py`.

Reply:
From my point of view, I see this as a more organized organization of folders of files. I lean towards _always_ creating a dedicated folder for the step.

This way, I know that whatever I find within a folder is for one and only one step. The other way around, I may have a mixture of files and folders in the same directory. I find that more confusing.

Reply:
OK, I agree. Usually it's useful to keep folders by version because all code is related in one way or another, so you can share modules. But in this case, where it's separate papers, it may not make sense. That said, it's unlikely that you'll have two papers datasets on the same day.